style.css

```diff
@@ -33,6 +38,23 @@ header {
   margin-top: 4px;
 }
 
+.menu-cont {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 48px;
+  width: 375px;
+  border: 1px solid white;
+}
+
+.menu-icon {
+  margin-right: 10px;
+}
+
+.dsktp-lf-menu {
+  display: none;
+}
```

Comment:
- [ ] The mobile icon is included in this class, so kindly separate the icon to be visible in the mobile version or may use it like the example below.

Example
```
.destp-if-menu .mobile-icon-menu {
 display: visible;
}